### DISCUSSION TITLE
Add SSH Timeout Setting

### DISCUSSION THREAD

**Discussion by benjamin74** [spatie/ray#617] [Ideas]
For some reason some of my servers are slow responding this morning and, whereas I can SSH with the terminal (takes about 9 seconds), I can't use Ray whose timeout is probably like 5 seconds.

I couldn't find any settings to configure the timeout so maybe it could be useful to have one.

[EDIT: ignore this, I restarted my modem and it's working fine]

**Comment by AdrianMrn** [comment]
The timeout is indeed set to 5 seconds. Let me know if this is something you'd like to be configurable, and I can look into it!
